config/routes.rb

```diff
@@ -23,4 +25,6 @@
   end
 
   resource :locale, only: [:update]
+
+  mount Sidekiq::Web => "/sidekiq"
```

Comment:
@hengsokly please prevent to let only the primary admin(aka system admin) can access to sidekiq web-ui dashboard